mp4parse/src/lib.rs

```diff
@@ -763,6 +776,7 @@ pub struct MediaContext {
     pub mvex: Option<MovieExtendsBox>,
     pub psshs: TryVec<ProtectionSystemSpecificHeaderBox>,
     pub userdata: Option<Result<UserdataBox>>,
+    pub metadata: Option<Result<MetadataBox>>,
```

Comment:
```suggestion
    #[cfg(feature = "meta-xml")]
    pub metadata: Option<Result<MetadataBox>>,
```
This field is only used with the code added in this PR, so I think it belongs behind the feature flag as well.